src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
```

Comment:
PullProofAI

The `UpdatePracticesDto` class looks well-defined with proper validation using `class-validator`. The `ApiProperty` decorator from `@nestjs/swagger` provides clear documentation for the API. This hunk is well-structured and adheres to best practices.

Overall, this code snippet is well-written and follows best practices for defining a DTO class in a NestJS application.

```diff
No changes needed. 
```

If you have any more code snippets or specific concerns for review, feel free to provide them!


<!-- This is an auto-generated comment by PullProofAI -->